src/content/blog/2024-10-17-the-data-process-that-powers-elections-in-utah.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: The Data Process That Powers Elections in Utah
+author: Miriam Seely
+description: >-
+  As you prepare for election day this year, learn more about how geospatial
+  data impacts where, when, and how you vote in Utah.
+date: '2024-10-17T16:31:16.697Z'
+category: UGRC
+cover_image: /src/images/pillar-blog/default-social-card.png
+cover_image_alt: ugrc social card
+---
+
+import Contacts from '@components/page/Contacts.astro';
+
+With the cool winds blowing in and the birds flying south, fall is officially upon us. That means a few things: warm sweaters, yellow aspen trees, and everyone's favorite method of acquiring a free sticker: voting! Election Day will be on Tuesday, November 5th, 2024. If you aren't registered to vote, you can learn how over on [vote.utah.gov](http://vote.utah.gov). Once you're registered, you can either vote by mail or in person. In both cases, where you vote and how your vote is counted are determined by geospatial data maintained by UGRC and local government authorities.
+
+UGRC might not be able to make the beautiful autumn hues of red, orange, and yellow stay longer than their natural time, but we are charged with safekeeping the data that makes elections in Utah possible. An example of these data are the [digital polygon representations](https://gis.utah.gov/products/sgid/political/voter-precincts/) of voting precinct boundaries. Maintaining these boundaries requires collaboration, organization, and a great attention to detail. Part of [our mission](https://gis.utah.gov/about/) as geospatial leaders in Utah is to “ensure a high level of coordination among Utah GIS users and effective, efficient use of GIS resources.” And boy oh boy, does making elections happen demand a lot of coordination. So, let's take a moment and get to know what's involved with keeping the data side of democracy alive.
+
+All 29 counties in the state of Utah employ public servants who manage surveying, recording, and verifying their county boundaries and voting precincts. These individuals, often County Recorders and County Clerks, submit their data to UGRC many months before election day. Every county sends in slightly different datasets according to the needs and circumstances of that county. UGRC adjusts each incoming dataset to fit a standardized schema and ensure accuracy of the information. The completed boundaries are used in elections at all levels, including city, county, state, and federal elections.
```

Comment:
```suggestion
All 29 counties in the state have public servants, often the County Recorders and Clerks, in charge of surveying, recording, and verifying their county boundaries and voting precincts. Each county submits these datasets many months before election day. Their schemas often differ slightly to meet each county's unique needs, so UGRC adjusts them to a standardized schema, ensures their spatial accuracy, and adds them to the statewide datasets. The completed boundaries are then used in elections at all levels—city, county, state, and federal.
```

It could also be nice to have a paragraph before this explaining exactly what a precinct is and how it's used in elections. Then this paragraph explains how they flow from counties to us, and the next can explain the consequences of errors (double count, wrong count, not counted) or accurate datasets (voters can have confidence in the election).

Reply:
I like that idea. I'll add a bit more about what voting precincts are.